syncthing_gtk/wizard.py

```diff
@@ -343,6 +345,10 @@ def search(self):
 					# File exists and is executable, run it and parse
 					# version string from output
 					log.info("Binary found in %s", bin_path)
+					if path == '/snap/bin':
+						set_is_snap()
```

Comment:
You can just add new key [somewhere here](https://github.com/syncthing/syncthing-gtk/blob/886466470e2c590352e9f4ab5f6bf104311be9a6/syncthing_gtk/configuration.py#L45) and then do `self.config['syncthing_is_snap'] = True` here. Config is "global" in sense that you can create instantiate Configuration object anywhere and read settings from it.